requirements.txt

```diff
@@ -1,18 +1,17 @@
-boto3==1.9.182
-flask~=1.1.4
+boto3~=1.20.20
+flask~=2.0.2
 flask-compress~=1.10.1
 flask-cors~=3.0.10
 flask-dropzone~=1.6.0
-flask-monitoringdashboard~=3.1.0
-flask-sqlalchemy~=2.5.0
-matplotlib==3.0.3
-mysqlclient~=1.4.6
-numpy>=1.16.4
-pillow>=7.1.0,<8.3.2
+flask-monitoringdashboard~=3.1.1
+flask-sqlalchemy~=2.5.1
```

Comment:
FYI I believe these patch upgrades are unnecessary since we use the ~=. It indicates that all 3.1.X are upgradable

Reply:
For flask-monitoringdashboard in particular, 3.1.1 is needed because 3.1.0 does not have Flask 2.0 support. Is it recommended to remove all patch upgrades and make all the dependencies like `example~=2.5.0`?

Reply:
Nope, no changes are needed, I just wanted to be clear.